it/api/endpoints/lossClaim/list/v3/ListLossClaimsControllerISpec.scala

```diff
@@ -391,6 +568,13 @@ class ListLossClaimsControllerISpec extends V3IntegrationBaseSpec {
       serviceErrorTest(Status.INTERNAL_SERVER_ERROR, "SERVER_ERROR", Status.INTERNAL_SERVER_ERROR, InternalError)
       serviceErrorTest(Status.SERVICE_UNAVAILABLE, "SERVICE_UNAVAILABLE", Status.INTERNAL_SERVER_ERROR, InternalError)
       serviceErrorTest(Status.BAD_REQUEST, "INVALID_CLAIM_TYPE", Status.BAD_REQUEST, TypeOfClaimFormatError)
+
+      //tys errors
+      serviceErrorTest(Status.BAD_REQUEST, "INVALID_CORRELATION_ID", Status.INTERNAL_SERVER_ERROR, InternalError)
```

Comment:
Think it's probably better to have a 1 x pre-TYS Sequence and 1 x TYS sequence of errors and that way we only need to call the `serviceErrorTest` method once

Can also import `Status` so we don't need e.g. `Status.BAD_REQUEST` but just `BAD_REQUEST`

Reply:
Both comments also apply to `validationErrorTest` method below